Go Client/Go Board.h

```diff
@@ -11,7 +11,11 @@ class GoBoard
 	board ReturnGoBoard();
 	stone ReturnWhosMoveIsNext();
 	stone ReturnWhosMoveJustWent();
+	uint16_t ReturnDeadStones(stone stone);
```

Comment:
you don't put the word "Return" in front of every function that returns something - too much noise man! :)